mrp_bom_attribute_match/reports/mrp_report_bom_structure.py

```diff
@@ -43,3 +42,60 @@ def _get_bom_lines(self, bom, bom_quantity, product, line_id, level):
                     if isinstance(value, models.NewId):
                         component[key] = value.origin
         return components, total
+
+    def _get_bom_lines(self, bom, bom_quantity, product, line_id, level):
+        components = []
+        total = 0
+        for line in bom.bom_line_ids:
+            line_quantity = (bom_quantity / (bom.product_qty or 1.0)) * line.product_qty
+            if line._skip_bom_line(product):
+                continue
+            company = bom.company_id or self.env.company
+            product_id = bom.get_component_template_product(
```

Comment:
Shouldn't this function be `_get_component_template_product`? With a leading underscore.

Reply:
Could you share error text ?

Reply:
Ah yes right. Thanks!